PALS_to_csv.go

```diff
@@ -0,0 +1,338 @@
+				purposeCode = purposeCode[0:2]                                   // FC
+				currentPurposes = append(currentPurposes, purposes[purposeCode]) // Facility Management
+			}
+		}
+		// activities
+		for i, activityBool := range line[44:93] {
+			if activityBool == "1" && i+44 != 53 { // need to avoid 53 as it's FACTS activity not a real activity
+				activityCode := headers[i+44]                                           // BM Biomass
+				activityCode = activityCode[0:2]                                        // BM
+				currentActivities = append(currentActivities, activities[activityCode]) // Biomass
+			}
+		}
+		newPals = append(newPals, PalsEntry{
+			ProjectNumber:                     line[1],
+			ForestId:                          line[2],
+			ProjectName:                       line[3],
+			LmuActual:                         line[4],
+			LmuRegion:                         line[5],
+			LmuForest:                         line[6],
+			LmuDistrict:                       line[7],
+			ProjectStatus:                     line[8],
+			ProjectCreated:                    line[9],
+			CreatedFy:                         line[10],
+			DecisionId:                        line[11],
+			DecisionName:                      line[12],
+			InitiationDate:                    line[13],
+			InitiationFy:                      line[14],
+			DecisionSigned:                    line[15],
+			SignedFy:                          line[16],
+			SignerLastName:                    line[17],
+			SignerFirstName:                   line[18],
+			SignerTitle:                       line[19],
+			DecisionType:                      line[20],
+			DecisionAppealRule:                line[21],
+			ProjectNoticeAndCommentRegulation: line[22],
+			AppealedOrObjected:                line[23],
+			NoCommentsOrOnlySupport:           line[24],
+			Litigated:                         line[25],
+			FactsActivity:                     line[53],
+			Activities:                        strings.Join(currentActivities, ", "),
+			Purposes:                          strings.Join(currentPurposes, ", "),
+			UniqueProject:                     line[94],
+			ElapsedDays:                       line[95],
+			UniqueDecision:                    line[96],
+			Ongoing:                           line[97],
+			DistrictId:                        line[98],
+			RegionId:                          line[99],
+			DecisionLevel:                     line[100],
+			RegionName:                        line[101],
+			Forest:                            line[102],
+			CalendarYearSigned:                line[103],
+			CalendarYearInitiated:             line[104],
+			OverallCaseOutcome:                line[105],
+			CaseStatus:                        line[106],
+		})
+	}
+
+	// open file
+	csvFile, err := os.Create(path)
+	if err != nil {
+		log.Fatalf("failed creating file: %s", err)
+		return err
+	}
+	defer csvFile.Close()
+
+	// build column headers
+	writer := csv.NewWriter(csvFile)
+	writer.Write([]string{
```

Comment:
any way we can leverage the existing headers for this and just skip the activity?